RELEASE_NOTES.txt

```diff
@@ -2,6 +2,238 @@ Apache Ignite Release Notes
 ===========================
 
 
+Apache Ignite In-Memory Distributed Database 2.10.0
+-----------------------------------------------------------
+
+
+(!) WARNINGS:
+* The community accepted MVCC API discontinuation in future releases
+* Enabling/disabling WAL doesn't work properly on unstable topology and will be fixed soon
+* Spring Data moved to ignite-extension and will be released separately
+* Node.js, Python, PHP thin clients moved to a dedicated repository
+
+
+Ignite:
+* Added API for checking network connectivity between all nodes in a cluster
+* Added API to get segment sizes
+* Added InitialSize and MaxSize to DataRegion metrics
+* Added RebalancingPartitionsTotal metrics
+* Added TDE CLI management tool
+* Added TDE cache key rotation
+* Added a metric for processed keys when rebuilding indexes
+* Added an ability to check driver metrics in Cassandra store
+* Added cluster performance profiling
+* Added cluster performance tool to build cluster performance report
+* Added cluster snapshot security permissions
+* Added command to control.sh to get an arbitrary Metric
+* Added command to control.sh to get an arbitrary SystemView
+* Added control.sh API - schedule & cancel
+* Added control.sh API - status
+* Added core defragmentation functions
+* Added deadlock detection for cache entry reentrant locks
+* Added defragmentation JMX API for schedule/cancel/status
+* Added defragmentation parallelism implementation
+* Added events for snapshot operation state
+* Added explicit command.sh option to enable experimental commands
+* Added extension of tracing of communication socket write with number of sent bytes
+* Added fallback to full partition rebalancing in case historical supplier failed to read all necessary data updates from WAL
+* Added generic to maintenance mode feature
+* Added implementation readiness probe REST endpoint
+* Added new advanced heuristics for historical rebalance
+* Added new checkpoint metrics
+* Added new event firing before query execution
+* Added new event for node join validation failure
+* Added new index manipulation commands to control.sh
+* Added new metrics framework in ZookeeperDiscovery
+* Added new type of WAL records to track atomic updates on backup nodes
+* Added non markers checkpoint implementation
+* Added output IgniteSystemProperties via control.sh
+* Added special mode for maintenance of Ignite node. Employing Maintenance Mode for clearing corrupted PDS files
+* Added stopping warm-up to control.sh
+* Added support to graceful shutdown for ZookeeperDiscoverySpi
+* Added system view for binary metadata
+* Added system view for metastorage items
+* Added system view walker generator wrong sorts imports
```

Comment:
It's hard to get to the meaning of this. Probably we should paraphrase this one.

Reply:
Fixed.